src/components/Input/Input.jsx

```diff
@@ -0,0 +1,32 @@
+const Input = (props) => {
+	const type = props.type ? props.type : 'text';
+	const inputClass = props.inputClass;
+	const labelClass = props.labelClass;
+	const id = props.id;
+	const title = props.title;
+	const placeholder = props.placeholder;
+	const value = props.value;
+	const change = props.change;
```

Comment:
Use destructuring assignment syntax and default values

Reply:
fixed